TestModels/SimpleTypes/SimpleDouble/codegen-patches/rust/dafny-4.5.0.patch

```diff
@@ -0,0 +1,1762 @@
+diff --git b/TestModels/SimpleTypes/SimpleDouble/runtimes/rust/dafny_impl/src/implementation_from_dafny.rs a/TestModels/SimpleTypes/SimpleDouble/runtimes/rust/dafny_impl/src/implementation_from_dafny.rs
+index 0609bbe6..5a759dc2 100644
+--- b/TestModels/SimpleTypes/SimpleDouble/runtimes/rust/dafny_impl/src/implementation_from_dafny.rs
++++ a/TestModels/SimpleTypes/SimpleDouble/runtimes/rust/dafny_impl/src/implementation_from_dafny.rs
+@@ -1,7 +1,7 @@
+ #![allow(warnings, unconditional_panic)]
+ #![allow(nonstandard_style)]
+ pub use dafny_standard_library::implementation_from_dafny::*;
+-
++    
+ pub mod r#_simple_dtypes_dsmithydouble_dinternaldafny_dtypes {
+     #[derive(PartialEq, Clone)]
+     pub enum DafnyCallEvent<I: ::dafny_runtime::DafnyType, O: ::dafny_runtime::DafnyType> {
+@@ -108,10 +108,7 @@ pub mod r#_simple_dtypes_dsmithydouble_dinternaldafny_dtypes {
+     }
+ 
+     impl GetDoubleInput {
+-        pub fn value(
+-            &self,
+-        ) -> &::std::rc::Rc<super::r#_Wrappers_Compile::Option<::dafny_runtime::Sequence<u8>>>
+-        {
++        pub fn value(&self) -> &::std::rc::Rc<super::r#_Wrappers_Compile::Option<::dafny_runtime::Sequence<u8>>> {
```

Comment:
Changing the formatting is unnecessary but harmless, will be cleaned up once we stop patching in general later :)